setup.py

```diff
@@ -1,88 +1,19 @@
-from pathlib import Path
 import sys
 import setuptools
 from setuptools import find_packages, setup
-from setuptools.command.build_ext import build_ext
-import subprocess
 
 # These next three lines fetch and import numpy, which is needed for installation
 import setuptools.dist
 setuptools.dist.Distribution().fetch_build_eggs(['Cython>=0.15.1', 'numpy>=1.10'])
-import numpy
-
 
 # please indent the lists with one element per line
 # for convenience of version control tools
 
 CAPNP_V=14
```

Comment:
```suggestion
```
Can be deleted.

Reply:
resolved with https://github.com/IBM/graph2tac/pull/98/commits/34c9863b7f235a1d59a219edd880fe7a74e15295